app/modules/ws2812_effects.c

```diff
@@ -1005,7 +1001,7 @@ static int ws2812_effects_start(lua_State* L) {
     state->counter_mode_call = 0;
     state->counter_mode_step = 0;
     // set the timer
-    os_timer_setfn(&(state->os_t), ws2812_effects_loop, NULL);
+    os_timer_setfn(&(state->os_t), ws2812_effects_loop, (void*)L);
```

Comment:
Some more testing revealed that passing the lua_State this way does not work.
Trying to find a clean solution.
Any hints welcome!